book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] Kindly add `:rentals` so it can got and set outside this class.

  ```
  attr_accessor :title, :author, :rentals
  ```

Reply:
Certainly, I will to that, thank you for pointing that out 🥇